src/Microsoft.NetCore.Analyzers/UnitTests/Security/DoNotUseInsecureDeserializerBinaryFormatterWithoutBinderTests.cs

```diff
@@ -940,14 +943,13 @@ public object DeserializeWithDelegate(byte[] bytes)
     }
 }",
                 GetCSharpResultAt(21, 20, BinderNotSetRule, "object BinaryFormatter.Deserialize(Stream serializationStream, HeaderHandler handler)"),
-                GetCSharpResultAt(21, 20, BinderNotSetRule, "object BinaryFormatter.UnsafeDeserialize(Stream serializationStream, HeaderHandler handler)")
-                );
+                GetCSharpResultAt(21, 20, BinderNotSetRule, "object BinaryFormatter.UnsafeDeserialize(Stream serializationStream, HeaderHandler handler)"));
         }
```

Comment:
Other than this test not reliably passing (diagnostic ordering?), all the changes LGTM.

Reply:
Thanks. I'll stabilize this one before merging.

Reply:
If fixing the test/test framework will take time, can we skip the test and merge the PR?

Reply:
I'll get it fixed today

Reply:
Fixed by simply removing the last commit from this PR